src/ui/common/src/components/tables/virtualizedTable.tsx

```diff
@@ -0,0 +1,252 @@
+import { Box, Typography } from '@mui/material';
+import { styled, Theme } from '@mui/material/styles';
+import TableCell from '@mui/material/TableCell';
+import clsx from 'clsx';
+import * as React from 'react';
```

Comment:
Nit: We've been doing `import React from "react";` by default. Slightly cleaner to read.

Reply:
I think this just came from the example code that we brought over, but should be an easy fix to stay consistent.